docs/android/gradle.md

```diff
@@ -179,28 +195,27 @@ Your project may use the other proguard file name.
 More information about R8/ProGuard configuration can be found here:
 [https://developer.android.com/studio/build/shrink-code](https://developer.android.com/studio/build/shrink-code)
 
-## 6. Build your app
+## 7. Build your app
 Now you can build your app with Gradle, i.e.:
 ```bash
 ./gradlew assemble
 ```
 
-**Please label your build with the Plugin Version from above i.e.**  
-`MyApp_PSi-0.14.2.apk` 
-**so our AI can learn how its dynamic instrumentation is performing on the build.**
+Please label your build with the Plugin Version from above i.e. `MyApp_PSi-{{ android_release() }}.apk` 
+so our AI can learn how its dynamic instrumentation is performing on the build.
 
 
 ----
 
 
 ## Enabling the plugin by build type
 
-For plugin versions greater than **0.12.1**, 
-you can integrate Product Science pipeline into your gradle build 
-selectively apply the plugin to a given build type by adding a `productScience` block 
-at the top level of your `app/build.gradle` file. 
+For versions of the plugin exceeding **0.12.1**, you have the option to selectively integrate the productscience plugin into your Gradle build. 
+This can be achieved by applying the plugin only to specific build types. 
```

Comment:
Again, passive voice:
If you are using a `productscience` plugin version greater that **0.12.1**, you can selectively integrate the `productscience` plugin into your Gradle build.
You will do this by applying the plugin only to specific build types.